lib/config/pricing/src/index.js

```diff
@@ -69,13 +69,6 @@ const error = (reason) => ({
 const plan = (ppid, country, auth, cb) => {
   debug('Retrieving pricing plan %s', ppid);
 
-  if(!ppid || !country) {
-    debug('Undefined pricing plan id / country');
-    let eMessage = !ppid ? 'Undefined pricing plan id' : '';
-    eMessage += !country ? ' Undefined pricing country' : '';
```

Comment:
Completely missed that. Thanks for catching it! 

With the collector's flow changes, ppid and country would always be defined.

Will remove that and stash the commit.